<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugDialog</class>
 <widget class="QDialog" name="DebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="5">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="6">
    <widget class="QGroupBox" name="eqBox">
     <property name="title">
      <string>Equalizer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>User row</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="6">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>5k </string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>3k </string>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <widget class="QSlider" name="verticalSlider_6">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QSlider" name="verticalSlider_4">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QSlider" name="verticalSlider_3">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSlider" name="verticalSlider_2">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>1k </string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QSlider" name="verticalSlider_5">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="verticalSlider">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>400</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>250</string>
            </property>
           </widget>
          </item>
          <item row="3" column="7">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>8k</string>
            </property>
           </widget>
          </item>
          <item row="2" column="8">
           <widget class="QSlider" name="verticalSlider_7">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="8">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>10k</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="hsBox">
     <property name="title">
      <string>Howling suppression</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PHPR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PNPR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="pnprEnabled">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="paprSpin">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> dB</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>PAPR</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="paprEnabled">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ISMD</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="phprSpin">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> dB</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="pnprSpin">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> dB</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="ismdEnabled">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="ismdSpin">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> dB</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="phprEnabled">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" rowspan="2">
    <widget class="QGroupBox" name="psBox">
     <property name="title">
      <string>Pitch shift</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>coef</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>osamp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="psCoef"/>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="psOsamp"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="gateBox">
     <property name="title">
      <string>Gate</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="raiseSpinBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Raise TH</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="fallSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="attackSpinBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="holdSpinBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="releaseSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Fall TH</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Attach time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Hold time</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Release time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3" rowspan="2">
    <widget class="QGroupBox" name="nsBox">
     <property name="title">
      <string>Noise suppression</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="highRadioButton">
        <property name="text">
         <string>High</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mediumRadioButton">
        <property name="text">
         <string>Medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lowRadioButton">
        <property name="text">
         <string>Low</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="compressorBox">
     <property name="title">
      <string>Compressor</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="normalizeCheckBox">
        <property name="text">
         <string>Normalize</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Attack time</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="cAttackSpinBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="cDecaySpinBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Decay time</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="cRatioSpinBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Ratio</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Compression</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="cCompressionSpinBox"/>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="cThresholdSpinBox"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Threshold db</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Noise floor db</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="cNoiseSpinBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="peakCheckBox">
        <property name="text">
         <string>Peak</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
